tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "types": ["vitest/globals"],
+    "types": ["vitest/globals", "@testing-library/jest-dom"],
```

Comment:
where you getting type errors?

Reply:
This was added back when I was having failures make commits because all the viewings tests were failing. You sent a link that suggested this change. I can revert this and see if I still have issues.